ead.xml

```diff
@@ -0,0 +1,170 @@
+    <relatedmaterial encodinganalog="3.5.3">
+      <p>Related material can be found in the A.M. Hedderick fonds.</p>
+    </relatedmaterial>
+    <accessrestrict encodinganalog="3.4.1">
+      <p>Open</p>
+    </accessrestrict>
+    <otherfindaid encodinganalog="3.4.5">
+      <p>http://library.wlu.ca/specialcollections/findingaid/3307</p>
+    </otherfindaid>
+    <dsc type="combined">
+      <c level="series">
+        <did>
+          <unittitle encodinganalog="3.1.2">Biograhical</unittitle>
+          <unitid encodinganalog="3.1.1" countrycode="CA" repositorycode="ON00362">S714-1</unitid>
+        </did>
+        <odd type="publicationStatus">
+          <p>published</p>
+        </odd>
+      </c>
+      <c level="series">
+        <did>
+          <unittitle encodinganalog="3.1.2">Prose</unittitle>
+          <unitid encodinganalog="3.1.1" countrycode="CA" repositorycode="ON00362">S714-2</unitid>
+        </did>
+        <odd type="publicationStatus">
+          <p>published</p>
+        </odd>
+      </c>
+      <c level="series">
+        <did>
+          <unittitle encodinganalog="3.1.2">Poetry</unittitle>
+          <unitid encodinganalog="3.1.1" countrycode="CA" repositorycode="ON00362">S714-3</unitid>
+        </did>
+        <odd type="publicationStatus">
+          <p>published</p>
+        </odd>
+      </c>
+      <c level="series">
+        <did>
+          <unittitle encodinganalog="3.1.2">Reviews</unittitle>
+          <unitid encodinganalog="3.1.1" countrycode="CA" repositorycode="ON00362">S714-4</unitid>
+        </did>
+        <odd type="publicationStatus">
+          <p>published</p>
+        </odd>
+      </c>
+      <c level="series">
+        <did>
+          <unittitle encodinganalog="3.1.2">Music</unittitle>
+          <unitid encodinganalog="3.1.1" countrycode="CA" repositorycode="ON00362">S714-5</unitid>
+        </did>
+        <odd type="publicationStatus">
+          <p>published</p>
+        </odd>
+      </c>
+      <c level="series">
+        <did>
+          <unittitle encodinganalog="3.1.2">Personal papers</unittitle>
+          <unitid encodinganalog="3.1.1" countrycode="CA" repositorycode="ON00362">S714-6</unitid>
+        </did>
+        <odd type="publicationStatus">
+          <p>published</p>
+        </odd>
+      </c>
+      <c level="series">
+        <did>
+          <unittitle encodinganalog="3.1.2">Business papers</unittitle>
+          <unitid encodinganalog="3.1.1" countrycode="CA" repositorycode="ON00362">S714-7</unitid>
+        </did>
+        <odd type="publicationStatus">
+          <p>published</p>
+        </odd>
+      </c>
+      <c level="series">
+        <did>
+          <unittitle encodinganalog="3.1.2">Publications</unittitle>
+          <unitid encodinganalog="3.1.1" countrycode="CA" repositorycode="ON00362">S714-8</unitid>
+        </did>
+        <odd type="publicationStatus">
+          <p>published</p>
+        </odd>
+      </c>
+    </dsc>
+  </archdesc>
+</ead>
```

Comment:
Is this XML included intentionally?

Reply:
Oops, my bad.  I was rushing to commit this last night. :(